open_event/helpers/data.py

```diff
@@ -478,14 +478,14 @@ def remove_microlocation(microlocation_id):
         flash('You successfully delete microlocation')
 
     @staticmethod
-    def create_user(form):
-        user = User(nickname='asdf',
-                    email=form['email'])
+    def create_user(userdata):
+        user = User(nickname='afwf',
```

Comment:
@aditya1702 @rafalkowalski @mariobehling I think we should remove the nickname field because it isn't being used anyways.. We are logging in using the email id only.